plat/rpi/rpi5/rpi5_svc_setup.c

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <common/debug.h>
+#include <common/runtime_svc.h>
+#include <drivers/scmi-msg.h>
+
+#include <rpi5_smc.h>
+
+/* Setup Standard Services */
+static int32_t rpi5_svc_setup(void)
+{
+	return 0;
+}
+
+/*
+ * Top-level Standard Service SMC handler.
+ */
+static uintptr_t rpi5_svc_smc_handler(uint32_t smc_fid, u_register_t x1,
+					  u_register_t x2, u_register_t x3,
+					  u_register_t x4, void *cookie,
+					  void *handle, u_register_t flags)
+{
+	uint32_t ret1 = 0U;
+
+	switch (smc_fid) {
+	case RPI5_SIP_SMC_SCMI:
+		scmi_smt_fastcall_smc_entry(0);
+		break;
+
+	default:
+		WARN("Unimplemented RPI5 Service Call: 0x%x\n", smc_fid);
+		ret1 = RPI5_SMC_NOT_SUPPORTED;
```

Comment:
It should be `SMC_UNC`:

```
The Unknown SMC Function Identifier is a sign-extended value of (-1) that is returned in the R0, W0 or X0
registers. An implementation must return this error code when it receives:
• An SMC or HVC call with an unknown Function Identifier
```

This is a difference between "unknown function" and "not supported function"

Reply:
Fixed.